api/src/prover/hypernova/mod.rs

```diff
@@ -0,0 +1,57 @@
+pub use super::nova::circuit;
+pub mod error;
+pub mod types;
+
+use std::path::Path;
+
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+
+use nexus_vm::VMOpts;
+
+use crate::prover::hypernova::{
+    error::ProofError,
+    types::{IVCProof, PP, SC},
+};
+
+use super::nova::Trace;
+
+pub fn save_proof<P: CanonicalSerialize>(proof: P, path: &Path) -> anyhow::Result<()> {
+    super::nova::save_proof::<P>(proof, path)
+}
+
+pub fn load_proof<P: CanonicalDeserialize>(path: &Path) -> Result<P, ProofError> {
+    super::nova::load_proof::<P>(path).map_err(ProofError::from)
+}
+
+pub fn run(opts: &VMOpts, pow: bool) -> Result<Trace, ProofError> {
+    super::nova::run(opts, pow).map_err(ProofError::from)
+}
+
+pub fn init_circuit_trace(trace: Trace) -> Result<SC, ProofError> {
+    super::nova::init_circuit_trace(trace).map_err(ProofError::from)
+}
+
+pub fn prove_seq(pp: &PP, trace: Trace) -> Result<IVCProof, ProofError> {
```

Comment:
test or small example would be a nice-to-have to avoid exposing something that potentially doesn't work

Reply:
Added, though let me know if I misunderstood what you were looking for.

Reply:
I meant an example similar to https://github.com/nexus-xyz/nexus-zkvm/blob/main/api/examples/prover_run.rs
maybe more concise just to showcase the usage of hypernova, will serve both as a test and a documentation for those who want to try it.

Reply:
I see, yes, will do.

Reply:
@slumber added